.github/workflows/run-checker-daily-sctp.yml

```diff
@@ -26,24 +24,24 @@ jobs:
         sudo apt-get update
         sudo apt-get -yq install lksctp-tools libsctp-dev
         checksctp
-        ${{ OK }} = $?
-        echo ${{ OK }}
+
     - name: Enable SCTP auth
+      id: sctp_auth
+      continue-on-error: true
       run: sudo sysctl -w net.sctp.auth_enable=1
-      if: $OK == 0
 
     - name: config
+      if: steps.sctp_auth.outcome == 'success' && steps.sctp_auth.conclusion == 'success'
```

Comment:
I'm not convince these are useful still.
A failure terminates the job.

Reply:
there is a continue-on-error..